src/java.base/share/classes/java/net/URLClassLoader.java

```diff
@@ -209,20 +202,10 @@ private synchronized void initializeSharedClassesSupport(URL[] initialClassPath)
      * @throws     NullPointerException if {@code urls} or any of its
      *             elements is {@code null}.
      */
-    @SuppressWarnings("removal")
     public URLClassLoader(URL[] urls, ClassLoader parent) {
         super(parent);
         initializeSharedClassesSupport(urls);                                    //OpenJ9-shared_classes_misc
-        this.acc = AccessController.getContext();
-        this.ucp = new URLClassPath(urls, null, this.acc, sharedClassServiceProvider); //OpenJ9-shared_classes_misc
-    }
-
-    URLClassLoader(String name, URL[] urls, ClassLoader parent,
-                   @SuppressWarnings("removal") AccessControlContext acc) {
-        super(name, parent);
-        initializeSharedClassesSupport(urls);                                    //OpenJ9-shared_classes_misc
-        this.acc = acc;
-        this.ucp = new URLClassPath(urls, null, this.acc, sharedClassServiceProvider); //OpenJ9-shared_classes_misc
+        this.ucp = new URLClassPath(urls, null, sharedClassServiceProvider); //OpenJ9-shared_classes_misc
```

Comment:
Please keep the comments lined up; suggest removing the extra spaces from line 207.
Similar comments for lines 227, 248, and 300.

Reply:
Updated